api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java

```diff
@@ -190,15 +142,79 @@ public CommandLineRunner run(
       final CustomMappingStrategy mappingStrategy = new CustomMappingStrategy();
       mappingStrategy.setType(WorkspaceExportRow.class);
 
-      StatefulBeanToCsv beanWriter =
-          new StatefulBeanToCsvBuilder(writer).withMappingStrategy(mappingStrategy).build();
+      try (FileWriter writer =
+          new FileWriter(opts.getOptionValue(exportFilenameOpt.getLongOpt()))) {
+        StatefulBeanToCsv beanWriter =
+            new StatefulBeanToCsvBuilder(writer).withMappingStrategy(mappingStrategy).build();
 
-      beanWriter.write(rows);
-
-      writer.close();
+        beanWriter.write(rows);
+      }
     };
   }
 
+  private WorkspaceExportRow toWorkspaceExportRow(DbWorkspace workspace) {
+    WorkspaceExportRow row = new WorkspaceExportRow();
+    row.setCreatorContactEmail(workspace.getCreator().getContactEmail());
+    row.setCreatorUsername(workspace.getCreator().getEmail());
+    row.setCreatorFirstSignIn(dateFormat.format(workspace.getCreator().getFirstSignInTime()));
+    row.setCreatorRegistrationState(workspace.getCreator().getDataAccessLevelEnum().toString());
+
+    row.setProjectId(workspace.getWorkspaceNamespace());
+    row.setName(workspace.getName());
+    row.setCreatedDate(dateFormat.format(workspace.getCreationTime()));
+
+    try {
+      row.setCollaborators(
+          extractAclResponse(
+                  workspacesApi.getWorkspaceAcl(
+                      workspace.getWorkspaceNamespace(), workspace.getFirecloudName()))
+              .keySet().stream()
+              .collect(Collectors.joining(",\n")));
+    } catch (ApiException e) {
+      row.setCollaborators("Error: Not Found");
+    }
+
+    Collection<DbCohort> cohorts = cohortDao.findByWorkspaceId(workspace.getWorkspaceId());
+    row.setCohortNames(
```

Comment:
I'd simply have a `row.setCohorts(Collection<DbCohort> cohorts)` and let the export row class be in charge of this formatting convention. It shortens this file a bit and allows you to support things like a Json field with a list of names, for example, without changing the logic here.